sentry-debug-images/src/integration.rs

```diff
@@ -0,0 +1,52 @@
+use std::borrow::Cow;
+
+use sentry_core::protocol::{DebugMeta, Event};
+use sentry_core::{ClientOptions, Integration};
+
+use crate::debug_images;
```

Comment:
_nit_: Rather than importing, can you just use the qualified path at call site? We usually don't import functions.